persistence/src/commonTest/kotlin/com/wire/kalium/persistence/dao/ConversationDAOTest.kt

```diff
@@ -858,6 +858,20 @@ class ConversationDAOTest : BaseDatabaseTest() {
         }
     }
 
+    @Test
+    fun givenObserConversationList_whenAConversationHaveNullAsName_thenItIsIncluded() = runTest {
```

Comment:
What do you think of cover as well the case of a conversation with `type=GROUP` + `name=null (maybe does not matter(` + `INCOMPLETE_METADATA=true` to expect to hide it :D

Reply:
done